mpa/modules/ov/models/mmcls/backbones/mmov_backbone.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import List, Dict
+
+from mmcls.models.builder import BACKBONES
+
+from ...mmov_model import MMOVModel
+from ....graph.parsers.cls.cls_base_parser import cls_base_parser
+
+
+@BACKBONES.register_module()
+class MMOVBackbone(MMOVModel):
```

Comment:
The `MMOVBackbone` class of the same name is implemented for each task. Could it be possible to unify this? 
If merging is not possible, consider renaming the individual classes..
Also, how about registering this as mmcv's backbone?
you can refer this [code](https://github.com/openvinotoolkit/training_extensions/blob/b19ab2f359ce10885d0f602f37890a70e321945e/otx/algorithms/common/adapters/mmcv/models/builder.py#L20),
maybe.. we can use `BACKBONES = Registry("models", parent=MMCV_MODELS, scope="omz")` and then we can use `omz.MMOVBackbone`

Reply:
First of all, we can't unify backbone since each library requires different outputs of backbone.
Could you explain why we need to name them differently?

I haven't seen, `omz.MMOVBackbone`, this kind of use case in OTX. Could you point it out?
What do you mean by **use omz.MMOVBackbone in model configuration.**?

Reply:
Renaming was for my suggestion about registering in one registry of mmcv.

Recently `feature/otx`, It has been modified to register and find backbones in the registry in the form of `otx.OTXMobileNetV3`, `mmdet.ResNet` or `torchvision.resnet`. It was just my suggestion for public backbone support.

However, leave this as it is in this PR for now, and see if I can unify the format. (according to the style of otx)